scripts/run-bazel-buildifier.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# INSTRUCTIONS:
+# This script runs a lint check on all the bazel files:
+# WORKSPACE
+# BUILD
+# *.bzl
+
+exec ./buildifier --lint=warn --mode=check --warnings=all -r app data domain model testing utility tools BUILD.bazel WORKSPACE oppia_android_test.bzl
```

Comment:
This expects the buildifier path to be the current path right? When we introduce pre-push hooks, that may not be the case so let's add a workspace path variable and prefix this call with that